dist/gmusic.js

```diff
@@ -195,6 +204,47 @@ proto.playback = {
     this.playback._sliderEl.dispatchEvent(evt);
   },
 
+  // Return song duration in milliseconds
+  getSongDuration: function () {
+    return this.playback._sliderEl.max;
+  },
+
+  getPlaybackState: function () {
+    // Play/Pause element states:
+    //   PLAYING: {__data__: {icon: 'av:pause-circle-filled'}, disabled: false}
+    //   PAUSED: {__data__: {icon: 'av:sj:pause-circle-fill'}, disabled: false}
+    //   STOPPED: {__data__: {icon: 'av:sj:play-circle-fill'}, disabled: true}
+    if (!this.playback._playPauseEl.disabled) {
+      if (this.playback._playPauseEl.__data__.icon === 'av:pause-circle-filled') {
+        return GMusic.Playback.PLAYING;
+      } else {
+        return GMusic.Playback.PAUSED;
+      }
+    } else {
+      return GMusic.Playback.STOPPED;
+    }
+  },
+
+  getSongInfo: function () {
+    var songInfo = {};
+    songInfo.title = this.doc.getElementById(SELECTORS.info.titleId).textContent;
+    songInfo.artist = this.doc.getElementById(SELECTORS.info.artistId).textContent;
+    songInfo.album = this.doc.querySelector(SELECTORS.info.albumSelector).textContent;
+    songInfo.art = this.doc.getElementById(SELECTORS.info.albumArtId);
+    songInfo.duration = this.doc.getElementById(SELECTORS.playback.sliderId).max;
```

Comment:
I think it was a miscommunication. When you said "remove #8", I interpretted it as deleting issue #8 (which we don't want). But I'm guessing you mean remove the `getSongDuration()` code/tests from this PR